docs/docs/cmd/spo/tenant/tenant-commandset-remove.md

```diff
@@ -0,0 +1,46 @@
+# spo tenant commandset remove
+
+Remove a ListView Command Set that is installed tenant wide.
+
+## Usage
+
+  ```sh
+spo tenant commandset remove [options]
+```
+
+## Options
+
+  `-t, --title [title]`
+: The title of the ListView Command Set. Specify either`title`, `id`, or`clientSideComponentId`.
+
+`-i, --id [id]`
+: The id of the ListView Command Set. Specify either`title`, `id`, or`clientSideComponentId`.
+
+`-c, --clientSideComponentId  [clientSideComponentId]`
+: The Client Side Component Id (GUID) of the ListView Command Set. Specify either`title`, `id`, or`clientSideComponentId`.
+
+--8 < -- "docs/cmd/_global.md"
+
+## Examples
+
+Removes an ListView Command Set by title.
+
+```sh
+m365 spo tenant commandset remove --title "Some command set"
+```
+
+Removes a ListView Command Set by id.
+
+```sh
+m365 spo tenant commandset remove --id 3
+```
+
+Removes a ListView Command Set by clientSideComponentId.
+
+```sh
+m365 spo tenant commandset remove --clientSideComponentId 7096cded-b83d-4eab-96f0-df477ed7c0bc
+```
+
+## Response
+
+The command won't return a response on success.
```

Comment:
Is the new line at the end of the file missing? Or is that just GH?

Reply:
It's there, just a GH thing
![image](https://github.com/pnp/cli-microsoft365/assets/35696168/375fb1ce-5de4-4b94-9a09-26ff73863cf1)